README.md

```diff
@@ -2,7 +2,8 @@
 
 WebView2Utilities help you develop and debug Microsoft Edge WebView2 apps.
 
-![Screenshot of WebView2 Utilities](screenshot.png)
+![image](https://github.com/david-risney/WebView2Utilities/assets/5773562/00846e42-6556-49c2-91ba-efffa45668ca)
```

Comment:
Please update the existing screenshot.png file in the repo rather than switching to a new thing.

I'm not sure where the assets image is or how its kept track of. The old image continues to exist in the git repo. The git repo is turned into the web page at https://david-risney.github.io/WebView2Utilities/ so it would be good to keep the image in the git repo as well to make sure its all in sync and there's one place to modify everything.